browser/js/catalog/product.js

```diff
@@ -3,11 +3,11 @@
 //state for single product
 app.config(function ($stateProvider) {
     $stateProvider.state('product', {
-        url: '/products/:productId',
-        templateUrl: 'js/catalog/products.html',
+        url: '/products/:id',
+        templateUrl: 'js/detail/templates/product.html',
```

Comment:
Are we reorganizing where we put code? Do we want to switch to the workshop style or maintain FSG practices?

I think we should be consistent and have explicit naming. If we keep the templates folder here, I suggest changing detail to productDetail or singleProduct so it's easier to find. 

There will be multiple "details" - user, review and product (at least)